### ISSUE TITLE
Heroku one-click deploy?

### ISSUE THREAD

**Issue by SachaG** [jsilland/txgh#5]
I know very little about how the whole thing works so maybe it's not applicable in this situation, but the other day I came across [Slackin](https://github.com/rauchg/slackin), which has a very convenient one-click deploy for Heroku.

So I just wanted to suggest maybe doing the same kind of thing for txgh. It could come in pretty handy :)

**Comment by matthewjackowski** [comment]
Hi @SachaG !  I just got a version of this project that runs on Heroku.  It's a bit more complicated since you need to upload your config settings to Heroku instead of a YAML file.  And you need to still setup web hooks.  I'm not sure exactly what you are looking for as far as a 'one-click' deploy.   I'm happy to discuss further if you are still interested.

**Comment by SachaG** [comment]
Thanks, but I ended up going with LingoHub over Transifex actually :)

**Comment by jdoconnor** [comment]
something like https://devcenter.heroku.com/articles/heroku-button along with a default-ish template with ENV variables may do the trick

**Comment by matthewjackowski** [comment]
@jdoconnor Excellent suggestion!  I'm going to clean up this repo a bit and make it easier to get up and running...and run tests for debug.  In the meantime I'll keep this issue open.

**Comment by ffmad** [comment]
hi ! Do you think you would be able to do that one click install ?

**Comment by matthewjackowski** [comment]
@ffmad Yes I think we are headed in a good direction for that.  Currently the tricky part is that we rely on a number of configuration files for the service to work.  And these files require you to add your own specific information...if we added 1-click right now, there would be a gap because you would not be able to provision a working system...you would still need a 2nd step to update those files.

If we can eliminate or greatly reduce that dependency within the 1-click process then I think it will work well.  I hope that makes sense!
